contributing/CONTRIBUTING.md

```diff
@@ -52,21 +52,9 @@ git commit -s
 
 #Install
 ```
-pip install .
+pip install sqlalchemy-ibmi
```

Comment:
This should be reverted. When you're developing and building from git, you will do pip install from the project directory instead of from what's on PyPI.